purchase_invoicing_no_zero_line/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,8 @@
+An automated task is included to invoice all pending purchase orders every
```

Comment:
* The word "automated task" make me think about scheduled job.
* Anything to do with "week"? I though this just remove the line when create invoice from PO.

Can you revise this?